src/tools/compiletest/src/runtest/run_make.rs

```diff
@@ -273,7 +264,13 @@ impl TestCx<'_> {
             p
         };
 
-        let mut rustc = Command::new(&self.config.rustc_path);
+        // run-make-support and run-make tests are compiled using the bootstrap compiler
+        let bootstrap_rustc = {
+            let mut p = build_root.join("stage0").join("bin").join("rustc");
```

Comment:
If you set the `rustc` key in `config.toml`, the compiler we use as bootstrap compiler doesn't get copied to `stage0/bin/rustc`.

Reply:
Yeah, figured that out a few moments before you posted it :laughing: I will just special case it for stage 0. It might not necessarily work for all cross-compilation configurations, but stage 0 tests are not fully supported anyway.